src/OSPSuite.UI/Views/Charts/ChartEditorView.cs

```diff
@@ -57,18 +60,51 @@ public ChartEditorView(IMenuBarItemToBarItemMapper barItemMapper, IImageListRetr
 
          _barEditItemForUsedIn.SuperTip = new SuperToolTip().WithText(ToolTips.UseSelectedCurvesToolTip);
 
+         _barEditItemLinkSimulationObserved = new BarEditItem(_barManager)
+         {
+            Edit = repositoryItemCheckEditLinkSimulationAndObserved,
+            Alignment = BarItemLinkAlignment.Right,
+            AutoFillWidth = false,
+            Width = 20,
+            Caption = Captions.LinkDataToSimulations,
```

Comment:
This might be too long for a caption with a width of 20,
Also beware of fixed width... I know this is how the other one is done, but we know that this bites you later on with different screen resolution. This might work...do not know

Reply:
I have tried it with different resolutions and it seems fine. I have also tried not specifying a fixed width, but it seems to break the alignment to the right, adding a small space, so I would leave it as is.